.gitignore

```diff
@@ -1,5 +1,4 @@
 bin/
-local/
```

Comment:
Can we keep this? I have almost 50 config files locally to test different scenarios and I'd have to move them out if we remove this. I don't know if anyone else uses this but it is quite harmless in any case. 🙏

Reply:
Not really, unless we change the location for that file

Reply:
Is either you change your local files or we change where the config is in examples

Reply:
Can we rename the ignore rule to `/local` so it only ignores the local dir at project root? If not, can we rename it to anything else? Let's say `local-etc` or something. I don't have a preferences for either as long as there is one gitignored directory in the project we can place helper config/scripts in without checking them in.

Reply:
@bogdandrutu thoughts? ^^^ Sounds useful to allow a place for ignored local config files.

Reply:
That can be achieve simply by having that in the local git config. I am not sure any other project uses that.